desktop/core/src/desktop/models.py

```diff
@@ -1785,7 +1785,8 @@ def get_config(self):
       'cluster_type': self.cluster_type,
       'has_computes': self.cluster_type in ('altus', 'snowball'), # or any grouped engine connectors
       'hue_config': {
-        'enable_sharing': ENABLE_SHARING.get()
+        'enable_sharing': ENABLE_SHARING.get(),
+        'collect_usage': COLLECT_USAGE.get()
```

Comment:
Nice!

Nit: Shall we add a comment giving some context that this is config is related to Google Analytics?

Reply:
Thanks! The config item is described in desktop/conf/pseudo-distributed.ini.tmpl